acceptance-tests/src/main/java/io/blueocean/ath/SmartWebElement.java

```diff
@@ -193,11 +188,9 @@ protected void sendInputEvent(WebElement el) {
      */
     private static void validateTextElement(WebElement element) {
         String tagName = element.getTagName().toLowerCase();
-        Preconditions.checkArgument(
-            "input".equals(tagName) || "textarea".equals(tagName),
-            "element must should be input or textarea but was %s",
-            tagName
-        );
+        if(!"input".equals(tagName) && !"textarea".equals(tagName)){
+            throw new IllegalArgumentException(String.format("element must be input or textarea but was %s", tagName));
+        }
```

Comment:
There are a bunch of these.  What about adding a `checkArgument()` method to one the blue ocean utils classes (or make a new one.  That way the local code remains mostly the same?

Reply:
this test module acceptance-tests do not use blueocean modules to avoid conflicts when testing. 
You must understand/see it as a separate project.

Reply:
Then why even have these checks? Why not assertions?

Reply:
no idea. maybe ask @@kzantow who created this in this commit https://github.com/jenkinsci/blueocean-plugin/commit/c2c513441b94219ea8495fda3d2a251417a4f4b2